bin/check_tron_jobs

```diff
@@ -115,30 +124,38 @@ def compute_check_result_for_job(client, job):
         job_content = client.job(tron_id.url, count=10, include_action_runs=True)
         results = compute_check_result_for_job_runs(job=job, job_content=job_content, client=client)
         kwargs.update(results)
+        log.info(kwargs["output"].split("\n")[0])
         return kwargs
 
 
-def check_job(job, client):
+def check_job(job, client, dry_run):
     if job.get('monitoring', {}) == {}:
-        print("Not checking {}, no monitoring metadata setup.".format(job['name']))
+        log.debug("Not checking {}, no monitoring metadata setup.".format(job['name']))
         return
     else:
-        print("Checking {}".format(job['name']))
+        log.info("Checking {}".format(job['name']))
         result = compute_check_result_for_job(job=job, client=client)
-        print(result)
-        send_event(**result)
-        print("")
+        if dry_run is True:
+            log.info("Would have sent this event to sensu: ")
+            log.info(result)
+        else:
+            log.debug(result)
+            send_event(**result)
 
 
 def main():
     options, args = parse_options()
     cmd_utils.setup_logging(options)
     cmd_utils.load_config(options)
     client = Client(options.server)
+    log.debug("DEBUG")
 
     jobs = client.jobs(include_job_runs=True)
     for job in jobs:
-        check_job(job=job, client=client)
+        if options.job is None:
+               check_job(job=job, client=client, dry_run=options.dry_run)
+        elif job['name'] == options.job:
```

Comment:
Seems like it should be possible to check one job without looping through all of them...

From a quick look at the client code, maybe you can use client.get_url(job_name) and then get the job with client.job or client.job_runs?